src/main/webapp/matches.html

```diff
@@ -3,11 +3,81 @@
   <head>
     <meta charset="UTF-8">
     <title>Friends of Friends</title>
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>Matches</h1>
+    <div class="container-fluid">
+      <nav class="navbar navbar-expand-sm justify-content-center navbar-background">
+        <ul class="navbar-nav">
+          <li class="nav-item">
```

Comment:
nit: rather than classing these all with `nav-item`, maybe apply the styling to `.navbar-nav li` (and similarly you could replace `.navbar-nav` in favor of `.nav ul`)

up to you though, this is just a matter of preference

Reply:
nav-item is a class from bootstrap that I am utilizing! same with navbar-nav

Reply:
ah makes sense!